gaptrain/gap.py

```diff
@@ -47,7 +52,7 @@ def ase_gap_potential_str(self):
         if not os.path.exists(f'{self.name}.xml'):
             raise IOError(f'GAP parameter file ({self.name}.xml) did not exist')
 
-        return ('pot = quippy.Potential("IP GAP", \n'
+        return (f'pot = {potential_class}("IP GAP", \n'
                 f'              param_filename="{self.name}.xml")')
```

Comment:
The indentation seems a bit extreme here? Is it aligned with the brackets, I can't tell?

Reply:
it's not quite right (and depends on the `potential_class`) but it's useful to have some indentation for readability. The file is only ever written to a temporary directory anyway